recipes/libcusparse/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "libcusparse" %}
+{% set version = "11.7.5.86" %}
+{% set platform = "linux-x86_64" %} # [linux64]
+{% set platform = "linux-ppc64le" %} # [ppc64le]
+{% set platform = "linux-sbsa" %} # [aarch64]
+{% set platform = "windows-x86_64" %} # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.tar.xz
+  sha256: 9250fe539d4bd6a378581dc0b528e8cfc418b57f28545bf39d70cae762075df7 # [linux64]
+  sha256: 1072e26dc118cbf9d6f061eddbff45f2da2eef6c87c2b8a64fd1586af91a2735 # [ppc64le]
+  sha256: 191ae1f26b15955b743f6c4978c8562b973b16671a9f684368d501919f906ce5 # [aarch64]
+  sha256: b51a46f4f6bb9476ffe433a1dedad2c94a546c8a92e70dfed63207b64ff57e50 # [win]
+
+build:
+  # uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # it is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # see https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  # if the installation is complex, or different between unix and windows, use separate bld.bat and build.sh files instead of this key.
+  # by default, the package will be built for the python versions supported by conda-forge and for all major oss.
+  # add the line "skip: true  # [py<35]" (for example) to limit to python 3.5 and newer, or "skip: true  # [not win]" to limit to windows.
+  # more info about selectors can be found in the conda-build docs: 
+  # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
+  #script: {{ python }} -m pip install . -vv
+  number: 0
+  skip: true  # [osx]
+  missing_dso_whitelist:
+    - /lib64/libpthread.so.0
+    - /lib64/librt.so.1
+    - /lib64/libdl.so.2
+    - /lib64/libm.so.6
+    - /lib64/libgcc_s.so.1
+    - /lib64/libc.so.6
+    - /lib64/ld-linux-x86-64.so.2
+
+outputs:
+  - name: libcusparse
+    files:
+      - lib/libcusparse.so.*
+    requirements:
+      run:
+        - libnvjitlink =12.*
```

Comment:
```suggestion
    requirements:
      host:
        - cuda-version {{ cuda_version }}
      run:
        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
        - libnvjitlink =12.*
```

Reply:
ditto, no need to pin `libnvjitlink` anymore?